ACE/ace/Log_Msg.h

```diff
@@ -161,6 +161,12 @@
 # undef THREAD
 #endif /* THREAD */
 
+#ifdef ACE_ANDROID
```

Comment:
Maybe add a ifndef ACE_DEFAULT_LOG_FLAGS around here so that the user can also set in his config.h file?

Reply:
Yes, I can